src/code/z_actor.c

```diff
@@ -1740,15 +1740,18 @@ void func_8002F974(Actor* actor, u16 sfxId) {
     actor->sfx = sfxId;
 }
 
-void func_8002F994(Actor* actor, s32 arg1) {
+void func_8002F994(Actor* actor, s32 timer) {
     actor->flags |= ACTOR_FLAG_28;
     actor->flags &= ~(ACTOR_FLAG_19 | ACTOR_FLAG_20 | ACTOR_FLAG_21);
-    if (arg1 < 40) {
-        actor->sfx = NA_SE_PL_WALK_DIRT - SFX_FLAG;
-    } else if (arg1 < 100) {
-        actor->sfx = NA_SE_PL_WALK_CONCRETE - SFX_FLAG;
+
+    // The sfx here are not actually sound effects, but instead this is data that gets sent into
+    // the io ports of sequence 0, and act as indices to a table to delay ticks of the timer sfx.
```

Comment:
since this comment is more user facing and doesnt require too much knowledge of the underlying workings of sfx, I would personally opt for a simpler comment like
"The sfx field is not used for an actual sound effect here. Instead, it controls the tick speed of the timer sound effect"

I just think the details are better described over in audio, for people who want to know how it works under the hood

Reply:
This is intended as a temporary comment until this is properly documented, to avoid the sfxId below being reverted to `NA_SE_`, but otherwise yeah

Reply:
i see. i dont think we would have to worry about an accidental reversion, the people who are reviewing things will be aware and catch it. but whatever

Reply:
I'm fine with the suggestion